lib/services/services.c

```diff
@@ -1356,3 +1359,27 @@ services__grab_stderr(svc_action_t *action)
     action->stderr_data = NULL;
     return output;
 }
+
+#if SUPPORT_SYSTEMD
+svc_action_t *
+services__systemd_get_inflight_op(const char *unit_name)
```

Comment:
Argh, this is so similar to `inflight_systemd()` but we don't have a `svc_action_t` here. I don't want to derail you in this PR, but as a wishlist item it might be nice to keep `inflight_ops` in a hash table since the order doesn't matter.

Reply:
https://projects.clusterlabs.org/T971